.drone.yml

```diff
@@ -38,16 +24,27 @@ pipeline:
   gulp-build:
     image: fidals/se-nodejs:dev
     environment:
-      - DEPS_DIR=/drone/deps
+      # deps from nodejs image
+      - DEPS_DIR=/usr/app/deps
     commands:
-      - cd /usr/app/src/
+      - cp -r /usr/app/src/node_modules .
       - gulp build
-      - cp -r front_build/ /drone/shopelectro
     when:
       event: [push, pull_request]
 
+  python-build:
+    image: docker/compose:1.22.0
+    commands:
+      - cd docker/
+      # Build python images with sources and static files
+      - docker-compose -f docker-compose-build.yml build python-dev python-prod
```

Comment:
>build python-dev python-prod

`python-prod` should be built on `push` event and  `python-dev` both on `push` and `pull_request`, so let's build them separately